entry_types/paged/lib/pageflow_paged/plugin.rb

```diff
@@ -2,7 +2,18 @@ module PageflowPaged
   # @api private
   class Plugin < Pageflow::Plugin
     def configure(config)
-      config.entry_types.register(PageflowPaged.entry_type)
+      config.entry_types.register(PageflowPaged.entry_type(config))
+    end
+
+    class PagedConfiguration
+      include Pageflow::Configuration::EntryTypeConfiguration
+
+      attr_accessor :page_types
```

Comment:
Doesn't this mean that there are now two page type collections - one in the main config and one here? Calling `config.page_types.register` outside of an `for_entry` block would add items to a different collection than calling it inside of the block. I think, the specs currently only pass since `Pageflow.config_for` is often called with a `PublishedEntry`/`DraftEntry` instead of an `Entry` which both do not delegate `type_name` at the moment. This causes `Pageflow.config_for` below to not return an entry type specific config.

I'd suggest to postpone this change to a later step when we also add the compat layer.